### DISCUSSION TITLE
Update Intervall

### DISCUSSION THREAD

**Discussion by Lucky10061992** [greghesp/ha-bambulab#535] [General]
Hi, 

I also connected a Magichome WRGB. However, I have the problem that when the chamber light is turned on, WRGB is only turned on 2 seconds later. This leads to the fact that the camera is then already calibrated to the previous brightness and then overexposed and not useable. It is clearly due to the update / pulling interval of this integration. Is there a possibility that the chamber light is queried every 500ms or even every 100ms? Where can I find this setting?

thank you for help 🤩

**Comment by AdrianGarside** [comment]
The integration doesn't poll. It reacts to data pushed from the printer as fast as it's provided it. Which is, at best at one second intervals, but the data provided after a printer state change has further delays before it gets included in the mqtt payload.

TLDR; no - the data is updated in the integration as fast as it is possible for it to be done.
